pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -385,6 +391,18 @@ type containerToKillInfo struct {
 	message string
 }
 
+// containerToUpdateInfo contains necessary information to update a container's resources.
+type containerToUpdateInfo struct {
+	// Value of the index of the specific container in pod.Spec.Containers.
+	specIndex int
```

Comment:
prefer storing a *v1.Container instead of the index.

Reply:
fixed.